src/seedu/addressbook/data/menu/Price.java

```diff
@@ -2,15 +2,16 @@
 
 import seedu.addressbook.data.exception.IllegalValueException;
 
+
 /**
  * Price of a particular menu item in the Restaurant Management System.
  * Guarantees: immutable; is valid as declared in {@link #isValidPhone(String)}
  */
 public class Price {
 
-    public static final String EXAMPLE = "440";
+    public static final String EXAMPLE = "$4.40";
     public static final String MESSAGE_PRICE_CONSTRAINTS = "Price should be in SGD and in dollars and cents";
-    public static final String PHONE_VALIDATION_REGEX = "\\d+";//"^\\$\\d+([.][0-9]+)?$";
+    public static final String PHONE_VALIDATION_REGEX = "\\$[1-9][0-9]*(\\.[0-9]{2})?|\\$0?\\.[0-9][0-9]";//"^\\$\\d+([.][0-9]+)?$";
```

Comment:
PRICE_VALIDATION_REGEX can be adjusted to accept price with only one digit after the dot